desktop/cmp/input/CodeInput.ts

```diff
@@ -326,22 +334,33 @@ class CodeInputModel extends HoistInputModel {
         if (this.cursor) this.clearSearchResults();
     };
 
-    onAutoFormat = () => {
-        if (!isFunction(this.componentProps.formatter)) return;
+    override toInternal(val: any) {
```

Comment:
Override fo the toInternal used by the `HoistInputModel`. Now will run the formatter if requested and properly configured.